packages/web/src/pages/gangsta-party.tsx

```diff
@@ -47,22 +50,51 @@ const GangstaParty = () => {
   const client = useOptimismClient();
   const { data, loading } = useAllHustlersQuery({ client });
```

Comment:
Are you sure the subgraph API doesn't have pagination built in? Can you double check? because if it does, we should fetch X hustlers each time and not fetch everything at once.